doc/tutorials/enable_s5.rst

```diff
@@ -160,21 +160,13 @@ The procedure for enabling S5 is specific to the particular OS:
 How to test
 ***********
 
-.. note:: The :ref:`CBC <IOC_virtualization_hld>` tools and service installed by
-   the `software-defined-cockpit
-   <https://github.com/clearlinux/clr-bundles/blob/master/bundles/software-defined-cockpit>`_ bundle
-   will conflict with the vUART and hence need to be masked.
+.. note:: Depends on the rootfs you used for SOS and UOS; system service maybe conflict with the vUART and hence need to be masked.
```

Comment:
I suspect @dbkinder will want to reword this, let's make sure we User VM and Service VM in the new wording.